deploy/example/mountstorage/README.md

```diff
@@ -0,0 +1,276 @@
+# Mount an azure blob storage
+
+In case you have the requirement, that your AKS cluster has to access a blob storage with kubelet identity or a dedicated user-assigned managed indentity, the following solution will do this.
+
+You can also use a differnt managed-identity for different persistent volumes (f.e. you have a pod, that should just have write access to some objects while having another pod, that should have write access everywhere.)
+
+
+## Before you begin
+
+- The Azure CLI version 2.37.0 or later. Run `az --version` to find the version, and run `az upgrade` to upgrade the version. If you need to install or upgrade, see [Install Azure CLI][install-azure-cli].
+
+- Install the aks-preview Azure CLI extension version 0.5.85 or later.
+
+- Ensure, that you are authenticated or run `az login` 
+
+- Run `az account set --subscription "mysubscription"` to select the right subscription
+
+- Create a storage account container and upload file, e.g.
+    ```bash
+    resourcegroup="aks-fuseblob-mi"
+    storageaccountname="myaksblob"
+    az storage account create -g "$resourcegroup" -n "$storageaccountname" --access-tier Hot  --sku Standard_LRS
+    az storage container create -n mycontainer --account-name "$storageaccountname" --public-access off
+    az storage blob upload \
+        --account-name myaksblob \
+        --container-name mycontainer \
+        --name test.htm \
+        --file test.htm \
+        --auth-mode key --account-key "$(az storage account keys list --account-name "$storageaccountname" --query '[0].value' -o tsv)"
+    ```
+
+## Using kubelet identity
+
+1. Give kubelet identity access to storage account
```

Comment:
option#1: grant kubelet identity access to storage account

Reply:
done